docs/source/how_tos/configure_workflow.rst

```diff
@@ -34,6 +37,9 @@ Here is an example of how to use the Picasso workflow with the inputs set:
                 STRAIN_PATH: redwood/base
                 SERVICE: mfe
                 ENABLE_LIMIT_BUILDKIT_PARALLELISM: false
+                RUNNER_WORKFLOW_LABEL: ubuntu-latest
```

Comment:
Should we include these inputs in the README?

Reply:
I did that here: https://github.com/eduNEXT/picasso/pull/37/commits/f186a2cb84b7d18b3d6759a141cdf2eb8220dc0a#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR91, thank you!